scripts/deploy/scripts/rescueFunds.ts

```diff
@@ -0,0 +1,144 @@
+import { config as dotenvConfig } from "dotenv";
+
+dotenvConfig();
+import {
+  ChainSocketAddresses,
+  DeploymentAddresses,
+  IntegrationTypes,
+  getAllAddresses,
+  networkToChainSlug,
```

Comment:
we need to take the addresses of older version here, how to do it? import from package? Can package version be taken as input?